index.html

```diff
@@ -8,152 +8,219 @@
             <header class="navbar">
                 <a href="#">RODCHAK</a>
+                <ul class="menu_bar desktop">
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
                 <nav><a href="#">
-                        <span class="material-symbols-outlined">menu</span></a>
+                        <span class="material-symbols-outlined mobile">menu</span></a>
                 </nav>
             </header>
-            <div class="content">
-                <h1>I’m Rodney <br> Glad to see <br> you!</h1>
-                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
-                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
-                    contact me.</p>
-                <h6><a href="#">LET’S CONNECT</a></h6>
-                <div class="social-media-icons">
-                    <ul class="social-media-icons">
-                        <li><a href="#"><img src="./images/Vector.svg" alt="twitter"></a></li>
-                        <li><a href="#"><img src="./images/Linkedin.svg" alt="Linkedin"></a></li>
-                        <li><a href="#"><img src="./images/medium.svg" alt="medium"></a></li>
-                        <li><a href="#"><img src="./images/github.svg" alt="github"></a></li>
-                        <li><a href="#"><img src="./images/angelist.svg" alt="angelist"></a></li>
-                    </ul>
+            <div class="background_content">
+                <div class="content">
+                    <h1>I’m Rodney <br> Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some
+                        of my
+                        work and experience! If you like what you see and have a project you need coded, don’t hestiate
+                        to
+                        contact me.</p>
+                    <h6><a href="#">LET’S CONNECT</a></h6>
+                    <div>
+                        <ul class="social-media-icons">
+                            <li><a href="#"><img src="./images/Vector.svg" alt="twitter"></a></li>
+                            <li><a href="#"><img src="./images/Linkedin.svg" alt="Linkedin"></a></li>
+                            <li><a href="#"><img src="./images/medium.svg" alt="medium"></a></li>
+                            <li><a href="#"><img src="./images/github.svg" alt="github"></a></li>
+                            <li><a href="#"><img src="./images/angelist.svg" alt="angelist"></a></li>
+                        </ul>
+                    </div>
                 </div>
+
             </div>
         </section>
         <section class="Works_Section">
             <div class="card card_1">
-                <img src="./images/Snapshot1.png" alt="Snapshoot_Portfolio_1">
```

Comment:
- [ ] Also, ensure that you add more padding for your work section to match the Figma design. I know you can do that :blush:

**1. Your card section:**
![Screenshot from 2022-07-09 14-24-03](https://user-images.githubusercontent.com/80612925/178106184-834e9386-6dea-412f-a5e7-eb0da0ea7f58.png)

**2. Figma design**
![Screenshot from 2022-07-09 14-40-34](https://user-images.githubusercontent.com/80612925/178106198-960a3f31-781d-4a17-b0ff-1b51ba096aea.png)